mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -45,7 +46,8 @@ private[regression] trait GeneralizedLinearRegressionBase extends PredictorParam
    * to be used in the model.
    * Supported options: "gaussian", "binomial", "poisson" and "gamma".
    * Default is "gaussian".
-   * @group param
+    *
```

Comment:
bad indentation here + a few places below.  My IntelliJ started doing this too, and I can't figure out how to prevent it...